src/Microsoft.AspNetCore.SignalR.Common/Protocol/HubMethodInvocationMessage.cs

```diff
@@ -45,6 +45,8 @@ protected HubMethodInvocationMessage(string invocationId, string target, object[
     /// </summary>
     public class InvocationMessage : HubMethodInvocationMessage
     {
+        public bool HasStream { get; }
```

Comment:
This for example is something that you'd need to update the HubProtocol spec with.